test/test_optim.py

```diff
@@ -1332,6 +1349,15 @@ def test_exp_lr(self):
         scheduler = ExponentialLR(self.opt, gamma=0.9)
         self._test(scheduler, targets, epochs)
 
+    def test_poly_lr(self):
```

Comment:
Perhaps it's worth adding a test that passes a list to the `min_lr` parameter. You will need multiple groups of parameters for that. See a similar Scheduler that does this at:
https://github.com/pytorch/pytorch/blob/14dba3d70208f5c6e4f1ea857c25d32a1d9d06d1/test/test_optim.py#L1497